### ISSUE TITLE
switchToIFrame missing else clause in 2.2.9 vendor\codeception\codeception\src\Codeception\Module\WebDriver.php

### ISSUE THREAD

**Issue by kpascal** [Codeception/Codeception#4045]
#### What are you trying to achieve?
my acceptance test is attempting to do a switchToIFrame() from 'iframe1' to 'iframe2'  and its blowing up on the switchToIFrame('iframe2');

#### What do you get instead?
I get an error :
invalid selector: An invalid or illegal selector was specified

> Provide console output if related. Use `-vvv` mode for more details.
invalid selector: An invalid or illegal selector was specified
 (Session info: chrome=56.0.2924.87)
 (Driver info: chromedriver=2.27.440174 
 Driver info: org.openqa.selenium.chrome.ChromeDriver
 Capabilities [{applicationCacheEnabled=false, rotatable=false, mobileEmulationEnabled=false, networkConnectionEnabled=false, chrome={chromedriverVersion=2.27.440174 
 *** Element info: {Using=css selector, value=frame#,iframe#} 

> Provide test source code if related

this is what the code looked like before 2.2.9
```
`public function switchToIFrame($name = null)
    {
        if (is_null($name)) {
            $this->webDriver->switchTo()->defaultContent();
        } else {
            $this->webDriver->switchTo()->frame($name);
        }
    }`
```

```
This is 2.2.9
`public function switchToIFrame($name = null)
    {
        if (is_null($name)) {
            $this->webDriver->switchTo()->defaultContent();
        }
        $this->webDriver->switchTo()->frame($name);
    }
`
```

When i make the changes locally i do not get any issues, it goes back to working

* Codeception version: 2.2.9
* PHP Version: 5.6.29
* Operating System: windows server 2008
* Installation type: Phar || Composer
* List of installed packages (`composer show`)
behat/gherkin                      v4.4.5 Gherkin DSL parser for PHP 5.3
codeception/codeception            2.2.9  BDD-style testing framework
doctrine/instantiator              1.0.5  A small, lightweight utility to instantiate objects in PHP without invo
facebook/webdriver                 1.3.0  A PHP client for WebDriver
guzzlehttp/guzzle                  6.2.2  Guzzle is a PHP HTTP client library
guzzlehttp/promises                v1.3.1 Guzzle promises library
guzzlehttp/psr7                    1.4.1  PSR-7 message implementation that also provides common utility methods
myclabs/deep-copy                  1.6.0  Create deep copies (clones) of your objects
phpdocumentor/reflection-common    1.0    Common reflection classes used by phpdocumentor to reflect the code str
phpdocumentor/reflection-docblock  3.1.1  With this component, a library can provide support for annotations via
phpdocumentor/type-resolver        0.2.1
phpspec/prophecy                   v1.6.2 Highly opinionated mocking framework for PHP 5.3+
phpunit/php-code-coverage          4.0.6  Library that provides collection, processing, and rendering functionali
phpunit/php-file-iterator          1.4.2  FilterIterator implementation that filters files based on a list of suf
phpunit/php-text-template          1.2.1  Simple template engine.
phpunit/php-timer                  1.0.9  Utility class for timing
phpunit/php-token-stream           1.4.11 Wrapper around PHP's tokenizer extension.
phpunit/phpunit                    5.7.14 The PHP Unit Testing framework.
phpunit/phpunit-mock-objects       3.4.3  Mock Object library for PHPUnit
psr/http-message                   1.0.1  Common interface for HTTP messages
psr/log                            1.0.2  Common interface for logging libraries
sebastian/code-unit-reverse-lookup 1.0.0  Looks up which function or method a line of code belongs to
sebastian/comparator               1.2.4  Provides the functionality to compare PHP values for equality
sebastian/diff                     1.4.1  Diff implementation
sebastian/environment              2.0.0  Provides functionality to handle HHVM/PHP environments
sebastian/exporter                 2.0.0  Provides the functionality to export PHP variables for visualization
sebastian/global-state             1.1.1  Snapshotting of global state
sebastian/object-enumerator        2.0.1  Traverses array structures and object graphs to enumerate all reference
sebastian/recursion-context        2.0.0  Provides functionality to recursively process PHP variables
sebastian/resource-operations      1.0.0  Provides a list of PHP built-in functions that operate on resources
sebastian/version                  2.0.1  Library that helps with managing the version number of Git-hosted PHP p
symfony/browser-kit                v3.2.4 Symfony BrowserKit Component
symfony/console                    v3.2.4 Symfony Console Component
symfony/css-selector               v3.2.4 Symfony CssSelector Component
symfony/debug                      v3.2.4 Symfony Debug Component
symfony/dom-crawler                v3.2.4 Symfony DomCrawler Component
symfony/event-dispatcher           v3.2.4 Symfony EventDispatcher Component
symfony/finder                     v3.2.4 Symfony Finder Component
symfony/polyfill-mbstring          v1.3.0 Symfony polyfill for the Mbstring extension
symfony/process                    v3.2.4 Symfony Process Component
symfony/yaml                       v3.2.4 Symfony Yaml Component
webmozart/assert                   1.2.0  Assertions to validate method input/output with nice error messages.

**Comment by DavertMik** [comment]
Thanks, this was fixed in #4001  but not released yet. Please get updates from 2.2 branch

**Comment by kpascal** [comment]
awesome thanks!
